2020/06/02/why-upgrading-software-libraries-is-imperative.html.md

```diff
@@ -0,0 +1,44 @@
+---
+author: "Selvakumar Arumugam"
+title: "Why Upgrading Software Libraries is Imperative"
+tags: software, version, upgrade
+gh_issue_number: 
+---
+
+Stack applications primarily consist of the Operating System as well as front and backend programming languages including their necessary list of library packages. The Operating System and programming language software can be periodically updated and run on the latest version. However, what about the long list of libraries being used in the application’s front and backend? As we know, It can be quite daunting to maintain and individually update a long list of software libraries. Still, it is important to keep them updated in several ways while mitigating the level of complexity. This post dives into our experience upgrading a complex stack with a list of libraries. What you will need, how to go about it, and the benefits of such an activity are all examined below. 
```

Comment:
Is "stack applications" a term people know? I haven't heard that before. It seems like we could just say "applications" since pretty much every application has a stack.

Reply:
@jonjensen You are right. Fixed it.